pxtpy/converter.ts

```diff
@@ -2937,4 +2937,37 @@ namespace pxt.py {
 
         return B.mkGroup(result);
     }
+
+    function unifyArrayType(e: Expr, fun: SymbolInfo, arrayType: Type) {
+        // Do our best to unify the generic types by special casing everything
+        switch (fun.qName!) {
+            case "Array.pop":
+            case "Array.removeAt":
+            case "Array.shift":
+            case "Array.find":
+            case "Array.get":
+            case "Array._pickRandom":
+                unifyTypeOf(e, arrayType.typeArgs![0]);
+                break;
+            case "Array.concat":
+            case "Array.slice":
+            case "Array.filter":
+            case "Array.fill":
```

Comment:
Should `Array.sort` be in here? It returns the input array https://github.com/microsoft/pxt/blob/master/libs/pxt-common/pxt-core.d.ts#L109

```suggestion
            case "Array.fill":
            case "Array.sort":
```

Reply:
It doesn't in Python